libnavigation-util/src/main/java/com/mapbox/common/NativeLoggerWrapper.kt

```diff
@@ -0,0 +1,23 @@
+package com.mapbox.common
+
+/**
+ * Wrapper on top of native logging functions that pipes information to Common SDK but also allows for unit testing the native functions.
+ * It has to live in a `com.mapbox.common` package to be able to access the package-private `com.mapbox.common.Log` class.
+ */
+internal object NativeLoggerWrapper {
```

Comment:
Can we reduce current logs path: logW -> LoggerFrontend -> NativeLoggerWrapper -> common. Maybe `NativeLoggerWrapper` should just implement `LoggerFrontend`?

Reply:
Unfortunately, I think this is necessary for us to be able to properly test the internal message formatting. If `NativeLoggerWrapper` implemented `LoggerFrontend`, did the formatting, and called native in the same function body, then we wouldn't be able to test it.